practice-one/index.html

```diff
@@ -6,502 +6,38 @@
-      <img class="objects-icon" alt="" src="assets/img/objects.svg" />
-           <!-- Footer -->
-      <div class="frame-parent26">
-        <div class="frame-parent14">
-          <div class="subscribe-to-our-newsletter-parent">
-            <div class="vector-container">
-              <img class="vector-icon" alt="" src="assets/img/vector1.svg" />
-
-              <div class="travlog">Travlog</div>
-            </div>
-            <div class="contrary-to-popular3">
-              Contrary to popular belief, Lorem Ipsum is not simply random text.
-              It has roots in a piece of classical Latin literature from 45 BC.
-            </div>
-          </div>
-          <div class="group-container">
-            <img class="group-icon2" alt="" src="assets/img/group2.svg" />
-
-            <img class="group-icon2" alt="" src="assets/img/group3.svg" />
-
-            <img class="group-icon2" alt="" src="assets/img/group-8.svg" />
-          </div>
-        </div>
-        <div class="frame-parent29">
-          <div class="company-parent">
-            <b class="company">Company</b>
-            <div class="about">About</div>
-            <div class="about">Career</div>
-            <div class="about">Mobile</div>
-          </div>
-          <div class="company-parent">
-            <b class="company">Contact</b>
-            <div class="about">Why Travlog?</div>
-            <div class="about">Partner with us</div>
-            <div class="about">FAQ’s</div>
-            <div class="about">Blog</div>
-          </div>
-          <div class="company-parent">
-            <b class="company">Meet Us</b>
-            <div class="about">+00 92 1234 56789</div>
-            <div class="about">info@travlog.com</div>
-            <div class="r-street-new-york-parent">
-              <div class="about">205. R Street, New York</div>
-              <div class="about">BD23200</div>
-            </div>
-          </div>
-        </div>
-      </div>
+    <!-- Start Heading -->
+    <div class="wrapper">
+      <header class="main__header">
+        <div class="header__top">
+        <div class="logo__header">
+            <a href="#">
+              <img class="logo__image" src="assets/img/vector.svg" alt="logo">
+            </a>
+            <span class="travlog">Travlog</span>
+          </div>
+          <nav class="header_navbar">
+            <ul class="nav_list">
+              <li class="nav_link"><a href="#">Home</a></li>
+              <li class="nav_link"><a href="#">Discovery</a></li>
+              <li class="nav_link"><a href="#">Special Deals</a></li>
+              <li class="nav_link"><a href="#">Contact</a></li>
+            </ul>
+          </nav>
+            <div class="button">
+              <button class="login__button">Log In</button>
+              <button class="signup__button">Sign Up</button>
+            </div>
+        </div>
+      </header>
```

Comment:
@quanghuy1799 Please don't apply the knowledge you still not training for now. The better way is to use write classes in a camel-cased syntax. For example, main-header instead of main__header (BEM is good but you still not gain it)